include/zephyr/fs/fuse_client.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2025 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Thanks for the rename. Could you also add a file comment telling a bit what this is meant to be used for?
Maybe something in the line of "This header provides helper functions to pack FUSE client requests. Note the client is the side which initiates these requests, and that in a typical FUSE usage the client would be the Linux kernel. While in Zephyr's case this is to enable functionality like the embedded virtiofs client connecting to a virtiofsd daemon running in the host." (or whatever content you think would help a future developer understand what they are looking at).